NGCHM/WebContent/javascript/NGCHM_Util.js

```diff
@@ -811,6 +811,9 @@ UTIL.getClickType = function (e) {
 	    case 3: clickType = 1;
 	    break; 
 	}
+	 if (e.ctrlKey) {
```

Comment:
No problem with this, but this function should be reviewed and rewritten or commented.